charter.html

```diff
@@ -158,14 +158,15 @@ <h2>Scope</h2>
             be implemented in browsers or similar user agents. The purpose of
             these features is to support web advertising without compromising
             user privacy.  Here "privacy" minimally refers to appropriate
-            processing of personal information.  Ways in which new features might
-            enable inappropriate processing include (but are not limited to)
-            enabling
+            processing of personal information that is collected by a user agent.  
+            Ways in which new features might enable inappropriate processing 
+            include (but are not limited to) enabling
             of <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-cross-site">cross-site
             or cross context recognition</a> of users or
             enabling <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-same-site">same-site
             or same-context recognition</a> of users across the clearing of
-            state.
+            state by transmitting a users’ data from a user agent that is within their control to an agent 
+            outside of their control with no guarantee of de-identification of the individual user.
```

Comment:
Not sure this is helpful in addressing the concern about ambiguity of processing, and I fear it introduces other ambiguities. User agents typically can't prevent all transmission of data to other actors -- making any request on the Web typically involves communicating something to another server, while using an IP address that makes it difficult to guarantee de-identification. Cross-context and same context recognition are more useful ideas than trying to explain in one sentence what data can or can't be transmitted.